src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/repository/ServiceCategoryRepositoryIntegrationTest.kt

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.service
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.ActiveProfiles
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.repository.ServiceCategoryRepository
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@ActiveProfiles("test", "local")
+class ServiceCategoryRepositoryIntegrationTest(@Autowired private val repository: ServiceCategoryRepository) {
+
+  @Test
+  fun `check entered`() {
+    val result = repository.findAll()
+    assertThat(result.count()).isEqualTo(9)
```

Comment:
this test will break if add more service categories. what are we actually trying to test?

Reply:
Was my initial test to ensure if it was all set ok.  I am not happy with with test anyway, so I will remove it.